### PR TITLE
Added option to grab created resource id from original data object.

### PR DESCRIPTION
**Pull request by jayalfredprufrock**
Maybe somebody can come up with a better option name??

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -160,8 +161,10 @@ class Service {
   }
 
   _create(data, params) {
-    return this.db().insert(data, this.id).then(rows => this._get(rows[0], params))
-      .catch(errorHandler);
+    return this.db().insert(data, this.id).then(rows => {
+      const id = this.idFromData ? data[this.id] : rows[0];
```
Comment (daffl) [inline src/index.js]: Does it need to be an option flag? I feel we could just fairly safely assume `const id = typeof data[this.id] !== 'undefined' ? data[this.id] : rows[0]`?

Comment (jayalfredprufrock) [inline src/index.js]: Yeah good call, an option is totally unnecessary

**Comment by daffl** [comment]
One comment and a tests for this would be sweet 😄

**Comment by jayalfredprufrock** [comment]
Cool I'll whip something up this week

**Comment by daffl** [comment]
Cool, let me know. I can get to it as well if you are too busy so that we can get this out in a new release.

**Comment by daffl** [comment]
Any news? I still think it would be a good fix with the changes we discussed.

**Pull request merged**